.github/actions/build-package/action.yml

```diff
@@ -60,15 +60,15 @@ runs:
       run: npm test
       working-directory: ${{ inputs.package }}
     - name: Pack artifacts (lib folder)
-      if: ${{ inputs.package == 'component-toolkit' || inputs.package == 'browser-test-tools' }}
+      if: ${{ inputs.package == 'component-toolkit' || inputs.package == 'browser-test-tools' || inputs.package == 'documenter' }}
```

Comment:
minor: since this is an expression (I suppose it is a javascript runner in GH actions?), can we simplify this for better readability?

`['component-toolkit', 'browser-test-tools', ...].includes(inputs.package)`

Reply:
No, this is not javascript: https://docs.github.com/en/actions/learn-github-actions/expressions

Reply:
This is GitHub's expression language and not javascript. This would be the closest you can get to this kind of expression: https://docs.github.com/en/actions/learn-github-actions/expressions#example-matching-an-array-of-strings

In this case I'm just following the pattern that's already there. Once all of the [pull requests](https://github.com/cloudscape-design/actions/pulls) for this are merged this this is no longer needed and will be removed and since all the existing pull requests already have this format I am not inclined to change it as I would need to update all the PRs so that they can be easily rebased.